composer.json

```
@@ -10,7 +10,7 @@
   ],
   "require": {
     "php": "^7.1||^8.0",
-    "friendsofphp/php-cs-fixer": "^2.7"
+    "friendsofphp/php-cs-fixer": "^2.7|^3.0"
```

```suggestion
    "friendsofphp/php-cs-fixer": "^3.0"
```

If you don't mind let's release major version. I don't see any forward compatibility layer for DefinedFixerInterface => FixerInterface change (fixer description will be not displayed in 2.x).

---

No problem. It's done @adamwojs.